src/heros/template/DefaultIFDSTabulationProblem.java

```diff
@@ -77,4 +77,9 @@ public int numThreads() {
 	public boolean computeValues() {
 		return true;
 	}
+	
+	@Override
+	public boolean recordEdges() {
+		return false;
```

Comment:
I think this is almost CERTAINLY the sane default: on any non-trivial analysis the graph gets inscrutable. It's pretty trivial to create anonymous subclasses of existing problems that enables edge recording (or maybe we can introduce a wrapper problem that enables debugging via delegation...)